ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -118,11 +118,8 @@ def _instantiate_profiler(self) -> bool:
                 return True
         # Catch any errors during profiling init and continue ingestion
         except Exception as exc:  # pylint: disable=broad-except
-            logger.error(
-                f"Error loading profiler {exc}"
-                "DataProfiler configuration is enabled. Please make sure you ran "
-                "pip install 'openmetadata-ingestion[data-profiler]'"
-            )
+            traceback.print_exc()
+            logger.error(f"Error loading profiler {repr(exc)}")
```

Comment:
let's update these to `logger.debug` so that these can be printed with `metadata --debug` commands